test.sql

```diff
@@ -1,22 +1,26 @@
+-- -*- mode: sql; product: postgres -*-
+
+-- Basic operations populating the database.
+
 CREATE FUNCTION unit_tests.create_hosts()
 RETURNS test_result AS $$
 DECLARE message test_result;
 DECLARE result boolean;
 DECLARE host_name text;
 BEGIN
     insert into hosts (name, data) values ('testbox', '{
-        "inet": ["192.0.2.4"],
-        "coordinates": {
-            "lat": 0,
-            "lon": 0
-        },
-        "location": "Undisclosed location",
-        "maxUsers": 1000
+	"inet": ["192.0.2.4"],
+	"coordinates": {
+	    "lat": 0,
+	    "lon": 0
+	},
+	"location": "Undisclosed location",
```

Comment:
btw, undisclosed location should just be `NULL`. perhaps make this `"Null Island"` instead?